src/main/java/Products.java

```diff
@@ -0,0 +1,59 @@
+public class Products {
```

Comment:
Look at proposed changes in attachment, tests could have to be adjusted to account for corrected totals
[Products.txt](https://github.com/Mekanation/tdd/files/4955267/Products.txt)